.github/workflows/ci.yaml

```
@@ -138,7 +136,7 @@ jobs:
 
       - name: Check that compiled Python dependency manifests are up-to-date with their sources
         # FIXME: There are issues related to testing with multiple Python versions.
-        if: ${{ startsWith(steps.set_up_python.outputs.python-version, '3.8.') }}
+        if: ${{ startsWith(matrix.python_version, '3.9.') }}
```

`matrix.python_version` does not end with a `.`, so this `if` would never be `true`.

```suggestion
        if: ${{ startsWith(steps.set_up_python.outputs.python-version, '3.9.') }}
```